src/Symfony/Bridge/Twig/Extension/SecurityExtension.php

```diff
@@ -27,6 +29,7 @@ final class SecurityExtension extends AbstractExtension
 {
     public function __construct(
         private ?AuthorizationCheckerInterface $securityChecker = null,
+        private ?UserAuthorizationCheckerInterface $userSecurityChecker = null,
         private ?ImpersonateUrlGenerator $impersonateUrlGenerator = null,
```

Comment:
```suggestion
        private ?ImpersonateUrlGenerator $impersonateUrlGenerator = null,
        private ?UserAuthorizationCheckerInterface $userSecurityChecker = null,
```
Otherwise it is a BC break.

Please also adjust the extension down below

Reply:
Good catch, thanks!